.github/workflows/opt-out-import-unit.yml

```diff
@@ -0,0 +1,20 @@
+name: opt-out-import unit tests
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/opt-out-import-unit.yml
+      - optout/**
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./optout
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+      - name: Run unit tests for opt-out-import lambda
+        run: go test
```

Comment:
This will need to be updated to run the full unit test.

Reply:
The lambda is encapsulated as part of the larger application (for now), so I don't think we need this section.

Reply:
Okay so the unit test workflow can be dropped for now?

Reply:
yep